org.claws_mail.Claws-Mail.json

```diff
@@ -0,0 +1,161 @@
+{
+  "app-id": "org.claws_mail.Claws-Mail",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.32",
+  "sdk": "org.gnome.Sdk",
+  "command": "claws-mail",
+  "finish-args": [
+    "--socket=x11",
+    "--socket=wayland",
+    "--share=ipc",
+    "--share=network",
+    "--filesystem=home:rw",
+    "--filesystem=/etc/skel/.claws-mail:ro"
```

Comment:
This one is odd. What are you trying to accomplish?

Reply:
Claws-Mail tries to access that location. I'm assuming for some template-configuration of maybe global mail-templates. That why I wanted to expose it as read-only, such that if this functionality is desired, it's at least not blocked.

Reply:
`/etc/skel` is meant to be used as skeleton files to be copied over by adduser/useradd for new users. Is anything broken without it? Personally I'd just drop it if it doesn't cause any errors.

Reply:
It doesn't even actually look at `/etc/`:

```
** (claws-mail:2): WARNING **: 11:19:59.753: failed to open directory: /app/etc/skel/.claws-mail
```